boot.sh

```diff
@@ -1,30 +1,37 @@
+#!/bin/bash
 # Boot up github shell access for Windows WSL/Ubuntu.
 
-#!/bin/bash
+if [ "$#" -lt 1 ]
+then
+echo  "Please insert at least one argument"
+exit
+fi
 
-private_key_location=~/.ssh/github_rsa # Change this location to where your file is located.
```

Comment:
Don't remove this yet, we can have a fallback.

Reply:
you want to keep both ways ie manually editing and optionally passing parameters?

Reply:
Yes. As mentioned above, the location parameter is optional.